sources/model/entity/Event.ts

```diff
@@ -52,4 +53,30 @@ export class Event {
             && this.calendar
             && this.startTime <= this.endTime;
     }
+
+    /**
+     * @brief Get an Event Instance by its ID
+     *
+     * @param conn
+     * @param event_id
+     */
+    public static getEventById(
+        conn: Connection,
+        event_id: number
+    ): Promise<Event> {
+        return conn.getRepository(Event)
+            .createQueryBuilder("event")
+            .innerJoinAndSelect("event.calendar", "calendar")
+            .where("event.id = :event_id", {event_id})
+            .getOne()
+    }
+
+    /**
+     * @brief Remove Event from calendar
+     *
+     * @param conn
+     */
+    public remove(conn: Connection): Promise<Event> {
+        return conn.getRepository(Event).remove(this);
+    }
```

Comment:
Make it a static function as in
https://github.com/PerfectWeek/web-api/blob/dev/sources/model/entity/Group.ts#L67
https://github.com/PerfectWeek/web-api/blob/dev/sources/model/entity/User.ts#L141

And name it `deleteById`